package.json

```diff
@@ -1,11 +1,34 @@
 {
-  "name": "webcomponents-jsonforms",
-  "private": true,
-  "version": "0.0.1",
+  "name": "jsonforms",
```

Comment:
Should we already use jsonforms as the name? I think we we'd need to update the existing docs for version 1 first, as `npm install` will fetch the latest version per default.